docker-compose.yml

```diff
@@ -12,17 +17,19 @@ services:
       - postgres
     depends_on:
       - postgres
-    restart: always
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:5057/_ping" ]
       interval: 30s
       timeout: 5s
       retries: 3
       start_period: 40s
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 2000M
     networks:
       - backend
-    environment:
```

Comment:
not used anywhere.

Reply:
PRODUCTION is not used anymore, other env variables may be used.